apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -375,6 +375,29 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
             if (unitType) return unitType
         }).filter(Boolean)
     }, [initialHasAllProperties, initialNewsItemScopes, initialProperties.length, initialSectionKeys])
+    const commonTemplates = useMemo(() => {
+        const commonTemplates = {}
+
+        for (const id in templates) {
+            if (templates[id].type === NEWS_TYPE_COMMON || isNull(templates[id].type)) {
+                commonTemplates[id] = templates[id]
+            }
+        }
+
+        return commonTemplates
+    }, [templates])
```

Comment:
Could you avoid using local `commonTemplates` that matches the same named constant in upper level scope.

For example:

```suggestion
    const commonTemplates = useMemo(() => {
        _.transform(templates, (result, value, key) => {
	    if (value.type === NEWS_TYPE_COMMON || isNull(value.type)) {
		result[key] = value
	    }
        }, {})
    }, [templates])
```